src/main/java/wellnus/reflection/ReflectionManager.java

```diff
@@ -0,0 +1,195 @@
+package wellnus.reflection;
+
+import wellnus.exception.BadCommandException;
+import wellnus.manager.Manager;
+
+import java.util.HashMap;
+import java.util.NoSuchElementException;
+
+public class ReflectionManager extends Manager {
+    private static final String FEATURE_NAME = "Self Reflection";
+    private static final String BRIEF_DESCRIPTION = "Users can get a random set of questions to reflect on.";
+    private static final String FULL_DESCRIPTION = "";
+    private static final String GET_COMMAND = "get";
+    private static final String GET_PAYLOAD = "";
+    private static final String RETURN_MAIN = "return";
+    private static final String RETURN_PAYLOAD = "";
+    private static final String EXIT_COMMAND = "exit";
+    private static final String EXIT_PAYLOAD = "";
+    private static final String NO_ELEMENT_MESSAGE = "There is no new line of input, please key in inputs.";
+    private static final String INVALID_COMMAND_MESSAGE = "Please check the available commands "
+            + "and enter a valid command.";
+    private static final ReflectUi UI = new ReflectUi();
+
+    // I need to set this as static if I want to set it to true if ExitCommand object.
+    // If I create another object in ExitCommand, the corresponding isExit will be for a new object,
+    // not the one we intend to terminate.
```

Comment:
Comments should not be in [first person](https://nus-cs2113-ay2223s2.github.io/website/schedule/week7/topics.html#implementation-code-quality-comments-basic-write-to-the-reader). Write to the reader